tests/remote-environment-controller-tests/README.md

```diff
@@ -0,0 +1,14 @@
+# Remote Environment Controller Tests
+
+## Overview
+
+This project contains the acceptance tests that you can run as part of the Kyma Application Connector testing process.
+The tests are written in Go. Run them as standard Go tests.
+
+## Usage
+
+This section provides information on building and versioning of the Docker image, as well as configuring the Kyma.
```

Comment:
`This section provides information on building and versioning` ~of~ `the Docker image, as well as configuring` ~the~ `Kyma.`

Reply:
Done